portfolio/src/main/webapp/index.html

```diff
@@ -6,7 +6,7 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
+  <body onload="getComments()">
```

Comment:
Is this the correct function name? I don't see a JavaScript function called getComments. Did you mean to use addGreeting?

Reply:
Yes, it is correct, but as you said in the beginning, I forgot to add the TODO statement.
I'll create the function and make another pull request for that one.